apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerConnectionStringBuilder.cs

```diff
@@ -50,10 +50,14 @@ public sealed class SpannerConnectionStringBuilder : DbConnectionStringBuilder
         private const string DataSourceKeyword = "Data Source";
         private const string UseClrDefaultForNullKeyword = "UseClrDefaultForNull";
         private const string EnableGetSchemaTableKeyword = "EnableGetSchemaTable";
+        private const string EmulatorHostVariable = "SPANNER_EMULATOR_HOST";
 
         private InstanceName _instanceName;
         private DatabaseName _databaseName;
 
+        private string _emulatorHost;
+        private Int32 _emulatorPort;
```

Comment:
nit: use `int` instead of `Int32`, `long` instead of `Int64`, `bool` instead of `Boolean`, etc.
Also, you can use `int? _emulatorPort`, `int?` represents the nullable int type.